### ISSUE TITLE
WKWebView crash

### ISSUE THREAD

**Issue by twodayslate** [twodayslate/claw#7]
[testflight_feedback.zip](https://github.com/twodayslate/claw/files/5273216/testflight_feedback.zip)

**Comment by twodayslate** [comment]
```
2020-09-23 23:23:39.548078-0400 claw[19764:18410166] [access] This app has crashed because it attempted to access privacy-sensitive data without a usage description.  The app's Info.plist must contain an NSPhotoLibraryAddUsageDescription key with a string value explaining to the user how the app uses this data.
```

**Comment by twodayslate** [comment]
fixed in 8cd813dc7062fa120c7cfbe190a3773999db06c3
